spec/wrappers/event_spec.rb

```diff
@@ -0,0 +1,24 @@
+########################################################################
+# event_spec.rb
+#
+# Test suite for the Azure::ArmRest::Event json wrapper class.
+########################################################################
+require 'spec_helper'
+
+describe "Event" do
```

Comment:
As custom methods get added to each class over time, having the separate spec files will be easier to deal with.